src/clients/dotnet/TigerBeetle.Tests/IntegrationTests.cs

```diff
@@ -1130,8 +1132,42 @@ public TBServer()
             format.WaitForExit();
             if (format.ExitCode != 0) throw new InvalidOperationException("format failed");
 
-            process = Process.Start(TB_SERVER, START);
+            process = new Process();
+            process.StartInfo.FileName = TB_SERVER;
+            process.StartInfo.Arguments = START;
+            process.StartInfo.RedirectStandardError = true;
+            process.Start();
             if (process.WaitForExit(100)) throw new InvalidOperationException("Tigerbeetle server failed to start");
+
+            var readPortEvent = new AutoResetEvent(false);
+            var addressLogged = "";
+            var stderrLogged = new StringBuilder();
+            process.ErrorDataReceived += (sender, e) =>
+            {
+                var line = e.Data;
+                if (line == null) return;
+                stderrLogged.Append(line);
+                var listening = "listening on ";

```

Comment:
Just a nit I noticed while trying to debug the mystery:

```suggestion
                const string listening = "listening on ";
```